scss/_settings_grid.scss

```diff
@@ -25,7 +26,18 @@ $grid-margin-widths: (
 ) !default;
 
 // Grid breakout template
+// TODO implement these with the new grid
 $l-fluid-breakout-max-width: $grid-max-width !default;
 $l-fluid-breakout-aside-width: 14rem !default;
 $l-fluid-breakout-main-child-width: 13rem !default;
 $l-fluid-breakout-3-col-threshold: calc($grid-max-width + $l-fluid-breakout-aside-width);
+
+// Legacy grid variables
```

Comment:
I know that I'm kind of going back and forth on it, so sorry for that.

But now, that we have this version, and see the scope of it, and the consequences of this change (based on the fact that there are other projects that use the old variables), would it make sense (to make the impact of this PR smaller), to keep the old variables with their old values, and introduce new variables for new values.

So instead of setting `$grid-columns` to new value of 8, and having new variable for the old value `$grid-12-columns` (set on 12), we would keep `$grid-columns: 12` (old value, for backwards compatibility), but introduce new `$grid-8-columns: 8` that we use internally for new grid code?

Would that simplify the migration? What do you think @jmuzina ?
Because I guess this will make our code a bit "dirtier" but it will keep the compatibility for others, right?

Reply:
Not sure if this comment is out-of-date now since [this thread](https://chat.canonical.com/canonical/pl/ejb5rdehxpg9met49yttofjwhr), but this should be a pretty simple change that reduces this PR's scope quite a bit, so I'll have a look and we can revert after if needed.

Reply:
Updated to expose two sets of variables - looks reasonable doable and is safer.